### PR TITLE
Added reporting variables

### PR DESCRIPTION
**Pull request by truthfool**
## Description

- added reporting variables
- Correction in accountHolding Institution Id values

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@fynmanoj

File: helm/superset/values.yaml [hunk 1]
```diff
@@ -472,8 +472,8 @@ ph-ee-engine:
         client_key: "0pLxbN83FrOl5Nd0Fh9Zi5BQlMxSL2n5"
         client_secret: "YzuGNoJxeub8ZC6d"
         pass_key: "bfb279f9aa9bdbcf158e97dd71a467cd2e0c893059b10f78e6b72ada1ed2c919"
+    paybill_accountHoldingInstitutionId: "gorilla"
```
Comment (fynmanoj) [inline helm/superset/values.yaml:475]: this wont work, it should be an env variable. please check and revert

**Review by fynmanoj — CHANGES_REQUESTED** [review]
please check and revert

File: helm/superset/values.yaml (hunk 1 above)
Comment (fynmanoj) [inline helm/superset/values.yaml:475]: please change this as per comments in env template

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Pull request closed without merging**